LEAF_Request_Portal/session_expire.php

```diff
@@ -0,0 +1,22 @@
+<?php
+// Include XSSHelpers
+if (!class_exists('XSSHelpers'))
+{
+    include_once __DIR__ . '/../libs/php-commons/XSSHelpers.php';
+}
+
+include '../libs/smarty/Smarty.class.php';
+
+function customTemplate($tpl)
```

Comment:
Need default value for $tpl attribute.  Additionally type hinting.  https://www.php.net/manual/en/language.types.declarations.php.  Would like to start seeing this both for the arguments and return type.  Exp:

function customTemplate(string $tpl='default_tpl_name')  ?string { ... }

Reply:
This has been updated in the latest commit.